sdk/communication/Azure.Communication.Common/tests/CommunicationTestEnvironment.cs

```diff
@@ -13,6 +13,8 @@ public class CommunicationTestEnvironment : TestEnvironment
         public const string AzurePhoneNumber = "AZURE_PHONE_NUMBER";
         protected const string TestPackagesEnabledDefaultValue = "all";
         private const string TestPackagesEnabledEnvironmentVariableName = "TEST_PACKAGES_ENABLED";
+        private const string SkipSmsTestEnvironmentVariableName = "COMMUNICATION_SKIP_SMS_TEST";
```

Comment:
where did you set these two variables? I don't find it in the `test.yml`

Reply:
and nit: maybe include `INIT` in the variable name so that people know this one is only applicable to INIT

Reply:
These variables are retrieved from 'subscription configuration'.  i.e. these settings in tests.yml.  The reason we have to switch to use 'subscription configuration' is we need to switch the values of these variables based on with cloud (Prod vs INT) the tests are running. 
```
         - $(sub-config-azure-cloud-test-resources)
          - $(sub-config-communication-services-cloud-test-resources-common)
          - $(sub-config-communication-services-cloud-test-resources-net)
    Clouds: Public
      Int:
        SubscriptionConfigurations:
          - $(sub-config-communication-int-test-resources-common)
          - $(sub-config-communication-int-test-resources-net)
```